collectors/ebpf.plugin/README.md

```diff
@@ -804,6 +804,29 @@ If they are already mounted, you will see an error. You can also configure your
 mount these filesystems on startup. More information can be found in
 the [ftrace documentation](https://www.kernel.org/doc/Documentation/trace/ftrace.txt).
 
+## Alerts
+
+Like other plugins, `eBPF.plugin` also have associated alerts for its charts. You can personalize them modifying
+the file `healt.d/ebpf.conf` using [`edit-config`](/docs/monitor/configure-alarms#edit-health-configuration-files).
+
+### Default alerts
+
+The following alerts are enabled by default:
+
+- retransmit_tcp_error: When at least 2% of packages are retransmitted, `netdata` raises alerts to notify possible
```

Comment:
I don't have us quoting "Netdata" in other areas (of this page and on learn). We can just say it in plain text.

Reply:
All right, I removed quotes and addressed everything you required. Thanks!